tabby/tools/converter/huggingface_gptneox_convert.py

```diff
@@ -145,8 +138,16 @@ def split_and_convert(args):
         "mlp.dense_4h_to_h.weight",
     ]
 
-    torch.multiprocessing.set_start_method("spawn")
-    pool = multiprocessing.Pool(args.processes)
+    huggingface_model_file_list = [__fn for __fn in os.listdir(args.in_file) if __fn.endswith(".bin")]
+    if len(huggingface_model_file_list) > 1:
```

Comment:
will checking `trained_gpu_num > 1`  be equivalent?

Reply:
I don't think there should be any difference in huggingface model files, whether trained on a single gpu or multiple gpus. The presence of multiple .bin files in the model files is due to the save rule implemented to prevent the model file from becoming too large. ([huggingface's documentation](https://huggingface.co/docs/transformers/v4.28.1/en/main_classes/model#transformers.PreTrainedModel.save_pretrained.max_shard_size))